tasks/25_rot13-encoder/rot13-encoder.js

```diff
@@ -0,0 +1,15 @@
+export const rot13Encoder = (str) => {
+  let newUtfNum = str.split('').map((cur) => cur.charCodeAt() + 13);
+  let rot13Arr = newUtfNum.map((cur) => {
+    if (cur >= 78 && cur <= 90) {
+      return String.fromCharCode(cur);
```

Comment:
maybe it's better to eject argument of `fromCharCode` and call this one at the end of this function.
it's more better to create separate function for getting char code by `cur` and then `map` callback will look like
```
const newCharCode = calculateCharCode(cur);
return String.fromCharCode(newCharCode);
```